doc/developing-packages.rst

```diff
@@ -4,52 +4,57 @@ Quickstart
 .. TIP::
     If this is your first time using `cabal` you should check out the `Getting Started guide <getting-started.html>`__.
 
-Let's assume we have created a project directory and already have a
-Haskell module or two.
+Starting from scratch, we're going to walk you through creating a simple
+Haskell application.
 
-Every project needs a name, we'll call this example "proglet".
+**TL;DR;** ``mkdir proglet && cd proglet && cabal init --simple --exe && cabal run proglet``
+
+
+Introduction
+------------
+
+Every application needs a name, we'll call ours "proglet" and start by
+creating an empty directory.
 
 .. highlight:: console
 
 ::
 
+    $ mkdir proglet
     $ cd proglet/
-    $ ls
-    Proglet.hs
 
-It is assumed that (apart from external dependencies) all the files that
-make up a package live under a common project root directory. This
-simple example has all the project files in one directory, but most
-packages will use one or more subdirectories.
 
-To turn this into a Cabal package we need two extra files in the
-project's root directory:
+Using ``cabal init``
+--------------------
 
--  ``proglet.cabal``: containing package metadata and build information.
+The ``cabal init`` command creates the necessary files for a Cabal package,
+it has both an ``--interactive`` (default) and ``--non-interactive``
```

Comment:
@ptkato: would you like to suggest edits or get involved in any other way?

Reply:
Not really, as far as I can tell the text goes through pretty smoothly.

Reply:
@ptkato: ok, great, thank you.

@m-renaud: that leaves us both. I hope this time it will stay `--interactive` by default for long. Which bit is particularly jarring to you? May I help? Again, apologies for the prolonged turn-around and conflicts.

Reply:
@m-renaud: oh, and feel free to take the text back to your original PR and tweak as needed there.

Reply:
Sorry, what I meant is that I thought that non-interactive is the default (opposite to what is claimed in the text), unless that has changed since version 3.2.

Reply:
I think `--interactive` is the default now and it's intended:

https://github.com/haskell/cabal/blob/master/cabal-install/src/Distribution/Client/Init.hs#L51-L99

Please review. :)